services/ymax-planner/src/plan-deposit.ts

```diff
@@ -107,6 +108,23 @@ export const getCurrentBalances = async (
   return currentBalances;
 };
 
+export const getNonDustBalances = async (
+  status: StatusFor['portfolio'],
+  brand: Brand<'nat'>,
+  powers: {
+    spectrum: SpectrumClient;
+    cosmosRest: CosmosRestClient;
+  },
+) => {
+  const currentBalances = await getCurrentBalances(status, brand, powers);
+  const nonDustBalances = Object.fromEntries(
+    Object.entries(currentBalances).filter(
+      ([, amount]) => amount.value > ACCOUNT_DUST_EPSILON,
+    ),
+  );
```

Comment:
Nit: consider using `objectMetaMap` (see [kernel.js](https://github.com/Agoric/agoric-sdk/blob/53e7ce5a2f103564f6c4155765d945ffb0368392/packages/SwingSet/src/kernel/kernel.js#L717) for an example):
```suggestion
  const nonDustBalances = objectMetaMap(currentBalances, desc =>
    desc.value > ACCOUNT_DUST_EPSILON ? desc : undefined,
  );
```

Reply:
That turns out to require additional type clutter, and I couldn't grok the documentation, so I'm leaving this for now.